packages/postgres-database/src/simcore_postgres_database/models/projects_tags.py

```diff
@@ -13,8 +13,13 @@
     sa.Column(
         "project_id",
         sa.BigInteger,
-        sa.ForeignKey(projects.c.id, onupdate="CASCADE", ondelete="CASCADE"),
-        nullable=False,
+        sa.ForeignKey(
+            projects.c.id,
+            onupdate="CASCADE",
+            ondelete="SET NULL",
+            name="project_tags_project_id_fkey",
+        ),
+        nullable=True,  # <-- NULL means that project was deleted
```

Comment:
maybe you want to adjust the doc below with this info instead?

Reply:
yes, please add it to the doc as well.